src/pages/Error/Page.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react';
+import { Main, PageHeader, PageHeaderTitle } from '@redhat-cloud-services/frontend-components';
+import { Messages } from '../../properties/Messages';
+import { EmptyStateSection } from '../../components/Policy/EmptyState/Section';
+import { ErrorCircleOIcon } from '@patternfly/react-icons';
+import { withRouter, RouteComponentProps } from 'react-router-dom';
+import { linkTo } from '../../Routes';
+import { Expandable, Text } from '@patternfly/react-core';
+import { join } from '../../utils/ComponentUtils';
+import { style } from 'typestyle';
+import { GlobalBackgroundColorDark300 } from '../../utils/PFColors';
+import { Spacer } from '../../utils/Spacer';
+
+type ErrorPageProps = RouteComponentProps<any>;
+
+interface ErrorPageState {
+    hasError: boolean;
+    error?: any;
+}
+
+interface ErrorStackProps {
+    error: any;
+}
+
+const errorClass = style({
+    fontFamily: 'monospace',
+    fontSize: '14px',
+    fontStyle: 'default',
+    textAlign: 'left',
+    backgroundColor: 'white',
+    borderWidth: 1,
+    borderStyle: 'solid',
+    borderColor: GlobalBackgroundColorDark300,
+    padding: Spacer.SM
+});
+
+const ErrorStack: React.FunctionComponent<ErrorStackProps> = (props) => {
+    const { error } = props;
+
+    if (error.stack) {
+        return (
+            <Text className={ errorClass } component="blockquote">
+                { join(error.stack.split('\n'), 'br') }
```

Comment:
With join, shouldn't 'br', instead be '<br/>' or <br></br>?

Reply:
'br' is fine. 
The argument is a component type (or name), when is a string, a raw html tag is used. 

Inside the code the passed component is used as `<TheComponent/>`.

Reply:
Thx. I need to go read up on  https://reactjs.org/docs/error-boundaries.html some more. Merging this to master.